chain-api/src/ethers/utils/utf8.ts

```diff
@@ -278,8 +280,15 @@ function getUtf8CodePoints(_bytes: BytesLike, onError?: Utf8ErrorFunc): Array<nu
  *
  *  If %%form%% is specified, the string is normalized.
  */
-export function toUtf8Bytes(str: string, form?: UnicodeNormalizationForm): Uint8Array {
-  assertArgument(typeof str === "string", "invalid string value", "str", str);
+export function toUtf8Bytes(str: string | BigNumber, form?: UnicodeNormalizationForm): Uint8Array {
+  assertArgument(
+    typeof str === "string" || BigNumber.isBigNumber(str),
+    `invalid string or bignumber value`,
+    "str or BN",
+    str
+  );
+
+  str = str.toString();
```

Comment:
actually we use `bn.toFixed()` as a serialization for BigNumber objects. My recommendation is to do the same here, to keep it more readable (since toString produces scientific notation for bigger numbers)

Reply:
Changed it out! I did wrap it so as not to change the string if it's passed in. I can remove the check if that's better